Added Argparse Usage; Config Usage; Readability; Enhanced with Class def; PEP8 syntax check

techcentaur opened the pull request:
Hey! I saw this code somewhere surfing through the tinder API and also used it to get my access token from Facebook. So, thanks for that :)

And, I thought since I used it, it is possible that someone else might want to use it. It would be better to make it more easy-to-use, readable, and easily mixable--overall, cool. :+1: 

So, I did the following thing
1. Added the argument parsing usage
2. Added a README.md and wrote usage in it
3. Added a configuration file for better encapsulation
4. Edited Class and definition variables
5. Added a requirements.txt file for installing modules all at once.

I hope you don't mind. It would be beneficial for others as well :+1:

techcentaur committed:
[ADD] readme.md for readability and doc

techcentaur committed:
[ADD] configuration file for easy uage

techcentaur committed:
[ADD] requirements file for easy installing of required modules

techcentaur committed:
[ADD] to ignore cached files

techcentaur committed:
[EDIT] argparse usage + class defined + config variables; overall enhanced

techcentaur committed:
[UPDATE] corrected syntax according to PEP8

AndrewPrifer reviewed (changes requested):
Looks good, although the fact that the script is only 30 lines was originally meant to be a feature in itself. 😅

Just one thing, are you sure you don't want to omit the way too specific `gavy42@jarvis:~/Desktop/tinder-token-retriever$` part from the readme file? I personally think just a `python3 get_token.py  -h` would be enough.

techcentaur committed:
[EDIT] removed computer name from readme.md

techcentaur commented:
Yeah, I understand that. :laughing: 

You are right, `python3 get_token.py -h` would be enough. Anyway, I have updated the readme file and removed the way too specific info, but I left the rest of the argparse usage. Thanks for the feedback. :+1: 

Does it seem okay?
